### PR TITLE
Adjust docs about Prow service accounts

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- adjust service account privileges in the docs

**Related issue(s)**

### CODE REVIEW

File: docs/prow/prow-installation-on-forks.md [hunk 1]
```diff
@@ -74,9 +74,9 @@ For the purpose of the installation, you must have a set of service accounts and
     - Compute OS Admin Login (`roles/compute.osAdminLogin`)
     - Service Account User (`roles/iam.serviceAccountUser`)
  - **sa-gcs-plank** with the role that allows the account to store objects in a bucket:
-    - Storage Object Admin (`roles/storage.objectAdmin`)
+    - Storage Object Admin (`roles/storage.objectAdmin`) in the `kyma-prow-logs` bucket
  - **sa-gcr-push-kyma-project** with the role that allows the account to push images to Google Container Repository in the `kyma-project` GCP project:
-    - Storage Admin (`roles/storage.admin`)
+    - Storage Object Creator (`roles/storage.objectCreator`) in `eu.artifacts.kyma-project.appspot.com` and `artifacts.kyma-project.appspot.com` buckets
```
Comment (klaudiagrz) [inline docs/prow/prow-installation-on-forks.md]:
```suggestion
    - Storage Object Creator (`roles/storage.objectCreator`) in the `eu.artifacts.kyma-project.appspot.com` and `artifacts.kyma-project.appspot.com` buckets
```

File: docs/prow/prow-installation-on-forks.md [hunk 2]
```diff
@@ -74,9 +74,9 @@ For the purpose of the installation, you must have a set of service accounts and
     - Compute OS Admin Login (`roles/compute.osAdminLogin`)
     - Service Account User (`roles/iam.serviceAccountUser`)
  - **sa-gcs-plank** with the role that allows the account to store objects in a bucket:
-    - Storage Object Admin (`roles/storage.objectAdmin`)
+    - Storage Object Admin (`roles/storage.objectAdmin`) in the `kyma-prow-logs` bucket
  - **sa-gcr-push-kyma-project** with the role that allows the account to push images to Google Container Repository in the `kyma-project` GCP project:
-    - Storage Admin (`roles/storage.admin`)
```
Comment (aszecowka) [inline docs/prow/prow-installation-on-forks.md]: It looks that there is a duplication in the documentation: https://github.com/kyma-project/test-infra/blob/master/docs/prow/authorization.md
@klaudiagrz, @piotrmiskiewicz can you fix it?

Comment (piotrmiskiewicz) [reply docs/prow/prow-installation-on-forks.md]: the docs are redesigned

File: docs/prow/authorization.md [hunk 3]
```diff
@@ -6,11 +6,12 @@ To deploy a Prow cluster, configure the following service accounts in the GCP pr
 
 | Service account name          | Usage                                                      | Required roles |
 | :---------------------------- | :----------------------------------------------------------| :------------- |
-| **sa-gcs-plank**              | Used by Prow plan microservice | `Storage Object Admin`
-| **sa-gke-kyma-integration**   | Running integration tests on GKE cluster | `Compute Admin`, `Kubernetes Engine Admin`, `Kubernetes Engine Cluster Admin`, `DNS Administrator`, `Service Account User`, `Storage Admin`
-| **sa-kyma-artifacts**         | Saving release artifacts to the GCS bucket | `Storage Object Admin`
-| **sa-vm-kyma-integration**    | Running integration tests on minikube | `Compute Instance Admin (beta)`, `Compute OS Admin Login`, `Service Account User`
-| **sa-gcr-push-kyma-project**  | Publishing docker images | `Storage Admin`
+| **sa-gcs-plank**              | Used by Prow plan microservice | `Storage Object Admin` (`roles/storage.objectAdmin`) in the `kyma-prow-logs` bucket
```
Comment (aszecowka) [inline docs/prow/authorization.md]: plan --> plank

File: docs/prow/authorization.md [hunk 4]
```diff
@@ -6,11 +6,12 @@ To deploy a Prow cluster, configure the following service accounts in the GCP pr
...
+| **sa-gke-kyma-integration**   | Running integration tests on GKE cluster | `Compute Admin` (`roles/compute.admin`), `Kubernetes Engine Admin` (`roles/container.admin`), `Kubernetes Engine Cluster Admin` (`roles/container.clusterAdmin`), `DNS Administrator` (`roles/dns.admin`), `Service Account User` (`roles/iam.serviceAccountUser`), `Storage Admin` (`roles/storage.admin`)
+| **sa-kyma-artifacts**         | Saving release artifacts to the GCS bucket | `Storage Object Admin` (`roles/storage.objectAdmin`)
```
Comment (aszecowka) [inline docs/prow/authorization.md]: not only release artifacts but also development artifacts

File: docs/prow/prow-installation-on-forks.md [hunk 5]
```diff
@@ -62,25 +62,16 @@ For the purpose of the installation, you must have a set of service accounts and
...
- - **kyma-bot-npm-token** which is a token for publishing npm packages
+ - **sa-gke-kyma-integration** with roles that allow the account to manage Kubernetes clusters and their resources.
+ - **sa-vm-kyma-integration** with roles that allow the account to provision virtual machines.
+ - **sa-gcs-plank** with the role that allows the account to store objects in a bucket.
+ - **sa-gcr-push-kyma-project** with the role that allows the account to push images to Google Container Repository in the `kyma-project` GCP project.
+ - **kyma-bot-npm-token** which is a token for publishing npm packages.
  - **sap-slack-bot-token** which is a token for publishing messages to the SAP CX workspace. Find more information [here](https://api.slack.com/docs/token-types#bot).
- - **sa-kyma-artifacts** service account with role that allows the account to store objects in a bucket:
-    - Storage Object Admin (`roles/storage.objectAdmin`)
+ - **sa-kyma-artifacts** service account with role that allows the account to store objects in a bucket.
```
Comment (klaudiagrz) [inline docs/prow/prow-installation-on-forks.md]:
```suggestion
 - **sa-kyma-artifacts** service account with the role that allows the account to store objects in a bucket.
```

File: docs/prow/authorization.md [hunk 6]
```diff
@@ -6,11 +6,12 @@ To deploy a Prow cluster, configure the following service accounts in the GCP pr
...
-| **sa-gcs-plank**              | Used by Prow plan microservice | `Storage Object Admin`
-| **sa-gke-kyma-integration**   | Running integration tests on GKE cluster | `Compute Admin`, `Kubernetes Engine Admin`, `Kubernetes Engine Cluster Admin`, `DNS Administrator`, `Service Account User`, `Storage Admin`
-| **sa-kyma-artifacts**         | Saving release artifacts to the GCS bucket | `Storage Object Admin`
-| **sa-vm-kyma-integration**    | Running integration tests on minikube | `Compute Instance Admin (beta)`, `Compute OS Admin Login`, `Service Account User`
-| **sa-gcr-push-kyma-project**  | Publishing docker images | `Storage Admin`
+| **sa-gcs-plank**              | Used by Prow plan microservice | `Storage Object Admin` (`roles/storage.objectAdmin`) in the `kyma-prow-logs` bucket
+| **sa-gke-kyma-integration**   | Running integration tests on GKE cluster | `Compute Admin` (`roles/compute.admin`), `Kubernetes Engine Admin` (`roles/container.admin`), `Kubernetes Engine Cluster Admin` (`roles/container.clusterAdmin`), `DNS Administrator` (`roles/dns.admin`), `Service Account User` (`roles/iam.serviceAccountUser`), `Storage Admin` (`roles/storage.admin`)
```
Comment (klaudiagrz) [inline docs/prow/authorization.md]:
```suggestion
| **sa-gke-kyma-integration**   | Runs integration tests on a GKE cluster. | `Compute Admin` (`roles/compute.admin`), `Kubernetes Engine Admin` (`roles/container.admin`), `Kubernetes Engine Cluster Admin` (`roles/container.clusterAdmin`), `DNS Administrator` (`roles/dns.admin`), `Service Account User` (`roles/iam.serviceAccountUser`), `Storage Admin` (`roles/storage.admin`)
```

File: docs/prow/authorization.md [hunk 7]
```diff
@@ -6,11 +6,12 @@ To deploy a Prow cluster, configure the following service accounts in the GCP pr
...
+| **sa-kyma-artifacts**         | Saving release artifacts to the GCS bucket | `Storage Object Admin` (`roles/storage.objectAdmin`)
+| **sa-vm-kyma-integration**    | Running integration tests on minikube | `Compute Instance Admin (beta)` (`roles/compute.instanceAdmin`), `Compute OS Admin Login` (`roles/compute.osAdminLogin`), `Service Account User` (`roles/iam.serviceAccountUser`)
```
Comment (klaudiagrz) [inline docs/prow/authorization.md]:
```suggestion
| **sa-vm-kyma-integration**    | Runs integration tests on Minikube. | `Compute Instance Admin (beta)` (`roles/compute.instanceAdmin`), `Compute OS Admin Login` (`roles/compute.osAdminLogin`), `Service Account User` (`roles/iam.serviceAccountUser`)
```

File: docs/prow/authorization.md [hunk 8]
```diff
@@ -6,11 +6,12 @@ To deploy a Prow cluster, configure the following service accounts in the GCP pr
...
+| **sa-gcr-push-kyma-project**  | Publishing docker images | `Storage Object Creator` (`roles/storage.objectCreator`) in the `eu.artifacts.kyma-project.appspot.com` and `artifacts.kyma-project.appspot.com` buckets
```
Comment (klaudiagrz) [inline docs/prow/authorization.md]:
```suggestion
| **sa-gcr-push-kyma-project**  | Publishes Docker images. | `Storage Object Creator` (`roles/storage.objectCreator`) in the `eu.artifacts.kyma-project.appspot.com` and `artifacts.kyma-project.appspot.com` buckets
```

File: docs/prow/authorization.md (hunk 3 above)
Comment (klaudiagrz) [inline docs/prow/authorization.md]:
```suggestion
| **sa-gcs-plan**              | Used by Prow plan microservice. | `Storage Object Admin` (`roles/storage.objectAdmin`) in the `kyma-prow-logs` bucket
```

File: docs/prow/authorization.md (hunk 4 above)
Comment (klaudiagrz) [inline docs/prow/authorization.md]:
```suggestion
| **sa-kyma-artifacts**         | Saves release and development artifacts to the GCS bucket. | `Storage Object Admin` (`roles/storage.objectAdmin`)
```

Comment (piotrmiskiewicz) [reply docs/prow/authorization.md]: done

File: docs/prow/authorization.md (hunk 3 above)
Comment (piotrmiskiewicz) [reply docs/prow/authorization.md]: done

File: docs/prow/authorization.md (hunk 4 above)
Comment (piotrmiskiewicz) [reply docs/prow/authorization.md]: done

File: docs/prow/prow-installation-on-forks.md (hunk 1 above)
Comment (piotrmiskiewicz) [reply docs/prow/prow-installation-on-forks.md]: not valid now

**Pull request merged**